src/app/mod.rs

```
@@ -455,7 +458,7 @@ impl Surreal {
 			Ok(range) => self.db.update(Resource::from(range.tb)).range((range.beg, range.end)),
 			Err(_) => self.db.update(Resource::from(resource)),
 		};
-		let data: Json = from_value(data)?;
+		let data = json(&from_value::<Json>(data)?.to_string()).unwrap_or(Value::None);
```

Same here. Can we please return the error, not fall back to `Value::None`?